xtable-service/src/main/java/org/apache/xtable/service/models/ConvertTableRequest.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+package org.apache.xtable.service.models;
+
+import java.util.List;
+import java.util.Map;
+
+import lombok.Builder;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@Builder
+public class ConvertTableRequest {
+  @JsonProperty("source-format")
+  private String sourceFormat;
+
+  @JsonProperty("source-table-name")
+  private String sourceTableName;
+
+  @JsonProperty("source-table-path")
+  private String sourceTablePath;
+
+  @JsonProperty("target-formats")
+  private List<String> targetFormats;
+
+  @JsonProperty("configurations")
```

Comment:
What are configurations? The term is overloaded let's use a different name?

Reply:
keeping this for now